src/patternfly/components/InputGroup/examples/InputGroup.md

```diff
@@ -40,14 +40,14 @@ cssPrefix: pf-c-input-group
 {{/input-group}}
 <br>
 {{#> input-group}}
-  {{#> dropdown id="dropdown-example-collapsed1" dropdown--HasToggleIcon="true"}}
-    {{#> dropdown-toggle-text}}
-      Dropdown
-    {{/dropdown-toggle-text}}
-  {{/dropdown}}
-  {{#> form-control controlType="input" input="true" form-control--attribute='type="text" id="textInput3" name="textInput3" aria-label="Input with dropdown and button" aria-describedby="inputDropdownButton1"'}}
+  {{#> select id="select-example-collapsed1" select--HasToggleIcon="true"}}
```

Comment:
Try this to create an inline style...

```suggestion
  {{#> select id="select-example-collapsed1" select--HasToggleIcon="true" select--attribute='style="width: 200px;"'}
```

Reply:
Thank you!  How does this look to you?

![Screen Shot 2020-04-08 at 18 38 30](https://user-images.githubusercontent.com/35377991/78840233-394d2a80-79c8-11ea-8527-c95ce8b0f884.png)

Reply:
Looks perfect! Does it render as 200px wide if you look at it in dev tools?

Reply:
It's showing up as 100px...

![Screen Shot 2020-04-08 at 19 34 39](https://user-images.githubusercontent.com/35377991/78843172-13c41f00-79d0-11ea-9ff0-08d30e494c2c.png)

Reply:
Oh OK, if you passed `select--attribute='style="width: 100px;"'` then that looks good. Can you verify that when you hover over the `.pf-c-select` element as you have in the screenshot on the right, that the blue box drawn in the browser shows that the select component is 100px? Feel free to submit the changes and I can verify that way.

Reply:
When I hover over the code on the right (`<div class="pf-c-select" style="width: 100px;">`), the tooltip on the left side of the screen, with the Select box highlighted, shows "div.pf-c-select 100 x 36".  But I have it set at 200px in my VS code file.

Reply:
Hmm, that's really odd! You might double check to make sure you've saved the file? And maybe try closing and re-opening? Sometimes I have to hit the circular refresh icon in the VS Code file explorer before it will show me changes to directories and filenames.

That said, 100px looks good though, so we can go with that if you want to submit a PR.

Reply:
It is odd, I've tried several things, and localhost isn't showing my changes correctly.  I'm going to commit these changes, with the width set to 100px.

Reply:
Looks good to me! I left a comment about `select--HasToggleIcon` - once that's updated we can merge this.